src/main/kotlin/uk/gov/gdx/datashare/controllers/EventsController.kt

```diff
@@ -77,7 +77,20 @@ class EventsController(
       "}\n" +
       "      </pre>" +
       "      <p><b>Mandatory Fields</b>: registrationDate, firstNames, lastName, sex, dateOfDeath</p>" +
-      "      <p><b>Gender Types:</b>: Male, Female, Indeterminate</p>",
+      "      <p><b>Gender Types:</b>: Male, Female, Indeterminate</p>" +
```

Comment:
```suggestion
      "      <p><b>Sex:</b>: Male, Female, Indeterminate</p>" +
```

I think since HMPO return `sex` as the field we should be conscious of the specific terminology

Reply:
Oh, or if HMPO return Gender then I think we should be returning that